blog/content/post/2020/02/14/rego_policies_opa.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Rego policies for Choria Server"
+date: 2020-02-14T09:00:00+07:00
+tags: ["security", "opa", "rego", "actionpolicy"]
+draft: true
```

Comment:
Do a update here, set draft false and add author like here:

https://github.com/choria-io/website/blame/b44bac23169b8aced9c724d586a5285dce625143/blog/content/post/2019/10/01/python_intro.md#L6